torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -2312,6 +2312,39 @@ def _call_function(
         )
 
 
+class PrintHigherOrderVariable(TorchHigherOrderOperatorVariable):
+    def _call_function(
+        self,
+        tx: "InstructionTranslator",
+        args: "list[VariableTracker]",
+        kwargs: "dict[str, VariableTracker]",
+    ) -> "VariableTracker":
+        from .builder import wrap_fx_proxy
+
+        if not isinstance(args[0], str):
+            unimplemented(
+                gb_type="Print: format printed with non string input not supported",
+                context=f"args: {args}",
+                explanation="torch._higher_order_ops.print only supports non string input in kwargs",
+                hints=[],
+            )
+        args, kwargs = LazyVariableTracker.realize_all((args, kwargs))
+
+        args_proxy = [arg.as_proxy() for arg in args]
+        kwargs_proxy = {k: v.as_proxy() for k, v in kwargs.items()}
+        return wrap_fx_proxy(
+            tx=tx,
+            proxy=tx.output.create_proxy(
+                "call_function",
+                self.value,
+                args=tuple(
+                    [self.script_obj_var.as_proxy(), self.method_name] + args_proxy
```

Comment:
why `[self.script_obj_var.as_proxy(), self.method_name]`? how are your test cases passes...

Reply:
You're right. The error doesn't show unless calling fullgraph so it didn't get caught. Good point I'll update this.